### PR TITLE
Adding Bagisto here.

### PR DESCRIPTION
**Pull request by papnoisanjeev**
Bagisto Will be great fit for up for grabs.

### CODE REVIEW

File: _data/projects/bagisto.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+name: Bagisto
+desc: Bagisto is a hand-tailored E-Commerce framework built on some of the hottest open-source technologies such as Laravel (a PHP framework) and Vue.js a progressive Javascript framework.
+site: https://bagisto.com/
+tags:
+  - opensource-ecommerce-framework
+  - laravel-ecommerce
+upforgrabs:
+  name: Easy
+  link: https://github.com/bagisto/bagisto/issues?q=is%3Aopen+is%3Aissue+label%3AEasy
```
Comment (ritwik12) [inline _data/projects/bagisto.yml]: @papnoisanjeev 
```suggestion
  link: https://github.com/bagisto/bagisto/labels/Easy
```

**Review by ritwik12 — CHANGES_REQUESTED** [review]

File: _data/projects/bagisto.yml (hunk 1 above)
Comment (papnoisanjeev) [reply _data/projects/bagisto.yml]: > @papnoisanjeev

@ritwik12  done.

**Comment by papnoisanjeev** [comment]
> This pull request will be closed tomorrow because it has not had recent activity after being reviewed. If you are still interested in adding your project to the site, please submit a new pull request and address the above feedback. Thank you for your contribution!

@ritwik12 
We have already done with suggested changes given by you [here](https://github.com/up-for-grabs/up-for-grabs.net/pull/3613/commits/d5aebcfc275ba6e3d8436ca0f0b341281fa26956).

If still anything need to be change please let us know.

**Comment by Silvrbckw** [comment]
papnoisanjeev:gh-pages

**Review by shiftkey — APPROVED** [review]

**Pull request merged**